src/Analysis/AtlasDataDictionary/AtlasLegacyDataDictionaryButton/AtlasLegacyDataDictionaryButton.css

```diff
@@ -0,0 +1,12 @@
+.atlas-data-dictionary-button {
```

Comment:
I believe this file is a duplicate of ```data-portal/src/Analysis/AtlasDataDictionary/AtlasDataDictionaryButton/AtlasDataDictionaryButton.css```

instead of duplicating it as a new file with identical code, could we just import the existing file into the new component? You might want to change the name and the directory structure as well so it is more clear the CSS is shared....for example

AtlasDataDictionaryButtons
----AtlasDataDictionaryButtons.css
----AtlasDataDictionaryButton [directory]
----AtlasDataDictionaryButtonLegacy [directory]

Reply:
I'll import the `AtlasDataDictionaryButtons.css` into the legacy one. The assumption is that the legacy button will be removed from code soon...so I don't want to change the regular button code.